### PR TITLE
Show CodeRed CMS version in wagtail admin

### PR DESCRIPTION
**Pull request by vsalvino**
Fixes issue #34 .

Replaced wagtail version in settings panel with CodeRed CMS version. Also added the version on hover of the logo in the bottom left. Did not replace the wagtail version hover on the main brand image, as that would have required forking an entire template and didn't seem worth it.

### CODE REVIEW

**Comment by corysutyak** [comment]
I like moving the version out of `setup.py`.  I would think it's necessary to fork the template for the wagtail version though.  It seems inconsistent to not show the same version everywhere.

**Comment by vsalvino** [comment]
Your right... I might fork the wagtail template in a separate file, add a block, and then extend that template, so it will be easy to diff/update the forked template with future wagtail updates.

File: coderedcms/templates/wagtailadmin/shared/menu_settings_menu_item.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "wagtailadmin/shared/menu_settings_menu_item.html" %}
+{% load coderedcms_tags %}
+
+{% block menu_footer %}
+    <li class="footer"><div class="menu-item"><p class="wagtail-version">CodeRed&nbsp;CMS v&nbsp;{% coderedcms_version %}</p></div></li>
+{% endblock %}
```
Comment (corysutyak) [inline coderedcms/templates/wagtailadmin/shared/menu_settings_menu_item.html]: Missing newline

**Comment by vsalvino** [comment]
Ok should be good to go.

**Pull request merged**